libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -337,6 +346,64 @@ describe("Drilling", () => {
+
+        it("Drilling down on chart two drillable attribute separates", () => {
+            Navigation.visit("dashboard/drilldown-on-chart");
+            const dateFilter = new DateFilter().open().selectAbsoluteForm();
+            new DateFilterAbsoluteForm()
+                .openFromRangePicker()
+                .typeIntoFromRangePickerInput("01/01/2011")
+                .openToRangePicker()
+                .typeIntoToRangePickerInput("12/31/2012");
+            dateFilter.apply().subtitleHasValue("01/01/2011 – 12/31/2012");
+            firstWidget
+                .scrollIntoView()
+                .waitChartLoaded()
+                .getChart()
+                .hasTooltipContents(0, ["Department", "Year (Closed)", "Amount"])
+                .clickSeriesPoint(0, 0);
+            drillModal
+                .selectDropdownAttribute(DIRECT_SALES)
+                .hasTitleHeader("Column with two drillable attributes › " + DIRECT_SALES)
+                .waitForDrillModalViz()
+                .hasTooltipContentsModal(0, ["Product", "Year (Closed)", "Amount"])
+                .close();
+            firstWidget.getChart().clickSeriesPoint(1, 0);
+            drillModal
+                .selectDropdownAttribute("2012")
+                .hasTitleHeader("Column with two drillable attributes › 2012")
+                .waitForDrillModalViz()
+                .hasTooltipContentsModal(0, ["Department", "Quarter/Year (Closed)", "Amount"]);
+        });
+
+        it("Drill down on chart two attribute same bucket with two drillable attributes", () => {
+            Navigation.visit("dashboard/drilldown-on-chart");
+            const bulletWidget = new Widget(1);
+            api.setUpDrillDownAttribute(DISPLAYFORM_PRODUCT, YEAR_CLOSE);
+            bulletWidget
+                .scrollIntoView()
+                .waitChartLoaded()
+                .hasTooltipContents(0, ["Department", "Product", "Amount"])
+                .clickSeriesPoint(0);
+            drillModal
+                .selectDropdownAttribute(DIRECT_SALES)
+                .hasTitleHeader("bullet with two attributes same bucket › " + DIRECT_SALES)
+                .waitForDrillModalViz()
+                .hasTooltipContentsModal(0, ["Product", "Amount"])
+                .clickSeriesPoint(0);
+            drillModal
+                .hasTitleHeader("bullet with two attributes same bucket › " + DIRECT_SALES + " › CompuSci")
+                .hasTooltipContentsModal(0, ["Year (Closed)", "Amount"])
+                .back();
+            bulletWidget.scrollIntoView().waitChartLoaded().getChart().clickSeriesPoint(0, 0);
+            drillModal
+                .selectDropdownAttribute("CompuSci")
+                .hasTitleHeader("bullet with two attributes same bucket › CompuSci")
+                .waitForDrillModalViz()
+                .hasTooltipContentsModal(0, ["Year (Closed)", "Amount"])
+                .close();
+            api.setUpDrillDownAttribute(DISPLAYFORM_PRODUCT);
```

Comment:
why do we need this step here?

Reply:
It will remove drilling from Product attribute. In somewhere, we don't use Product to drilling.